akamai/cache-client.go

```diff
@@ -79,9 +81,20 @@ func NewCachePurgeClient(
 	retries int,
 	retryBackoff time.Duration,
 	log blog.Logger,
-	stats metrics.Scope,
+	stats prometheus.Registerer,
 ) (*CachePurgeClient, error) {
-	stats = stats.NewScope("CCU")
+	purgeLatency := prometheus.NewHistogram(prometheus.HistogramOpts{
+		Name:    "ccu_purge_results",
+		Help:    "Histogram of latencies of CCU purges",
+		Buckets: metrics.InternetFacingBuckets,
+	})
+	purges := prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "ccu_purges",
+		Help: "A counter of CCU purges labelled by the result",
+	}, []string{"type"})
+	stats.MustRegister(purgeLatency)
```

Comment:
tiny nit: I think its better to group the `prometheus.NewXXX` invocations with their respective `stats.MustRegister` to make it easier to spot if one is missed.